### PR TITLE
LPS-88163 Client GET tests for Folders

### PR DESCRIPTION
**Pull request by ruben-pulido**

### CODE REVIEW

**Comment by ruben-pulido** [comment]
ci:test:sf <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by ruben-pulido** [comment]
ci:test:relevant <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/headless-apio/folder/folder-apio-client-test/build.gradle [hunk 1]
```diff
@@ -0,0 +1,16 @@
+dependencies {
+	testIntegrationCompile group: "com.jayway.jsonpath", name: "json-path", version: "2.2.0"
+	testIntegrationCompile group: "com.liferay.arquillian", name: "com.liferay.arquillian.arquillian-container-liferay", version: "1.0.7"
+	testIntegrationCompile group: "com.liferay.arquillian", name: "com.liferay.arquillian.arquillian-deployment-generator-bnd", version: "1.0.4"
+	testIntegrationCompile group: "com.liferay.portal", name: "com.liferay.portal.kernel", version: "default"
+	testIntegrationCompile group: "io.rest-assured", name: "rest-assured", version: "3.2.0"
+	testIntegrationCompile group: "org.apache.httpcomponents", name: "httpasyncclient", version: "4.1.3"
+	testIntegrationCompile group: "org.apache.httpcomponents", name: "httpclient", version: "4.5.3"
```
Comment (cgoncas) [inline modules/apps/headless-apio/folder/folder-apio-client-test/build.gradle]: DO we need this dependency?

File: modules/apps/headless-apio/folder/folder-apio-client-test/src/testIntegration/java/com/liferay/folder/apio/client/test/FolderApioTest.java [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+			"_links.content-space.href"
+		).follow(
+			"_embedded.ContentSpace.find { it.name == '" +
+				FolderTestActivator.CONTENT_SPACE_NAME + "' }._links." +
+					"documentsRepository.href"
+		).then(
+		).statusCode(
+			200
+		).body(
+			"_links.documents.href", IsNull.notNullValue()
```
Comment (cgoncas) [inline modules/apps/headless-apio/folder/folder-apio-client-test/src/testIntegration/java/com/liferay/folder/apio/client/test/FolderApioTest.java]: Documents are not another API?

Comment (cgoncas) [reply modules/apps/headless-apio/folder/folder-apio-client-test/src/testIntegration/java/com/liferay/folder/apio/client/test/FolderApioTest.java]: But the link is defined in folder, OK :)

File: modules/apps/headless-apio/folder/folder-apio-client-test/src/testIntegration/java/com/liferay/folder/apio/client/test/FolderApioTest.java [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+		return OAuth2ProviderTestUtil.getArchive(FolderTestActivator.class);
+	}
+
+	@Before
+	public void setUp() throws MalformedURLException {
+		_rootEndpointURL = new URL(_url, "/o/api");
+	}
+
+	@Test
+	public void testContentSpaceLinkExistsInRootEndpoint() {
```
Comment (cgoncas) [inline modules/apps/headless-apio/folder/folder-apio-client-test/src/testIntegration/java/com/liferay/folder/apio/client/test/FolderApioTest.java]: This test is included in content space, we don't need to test it here.

**Pull request closed without merging**